doc/JSON_INFO.md

```diff
@@ -1724,6 +1725,10 @@ Same as for terrain, see below in the chapter "Common to furniture and terrain".
 
 Movement cost modifier (`-10` = impassable, `0` = no change). This is added to the movecost of the underlying terrain.
 
+#### `light_emitted`
+
+How much light the furniture produces.
```

Comment:
what exactly does a value of "10" mean? 10 tiles? 10 lumens?

Reply:
I can't find any documentation on it, but I'll clarify it a bit.

Reply:
you could get away with a few examples... 10 is the brightness of a computer screen, 120 is an overhead light. I would say it's probably incandescent bulb watt equivalents.